Assets/scss/components/_newfooter.scss

```diff
@@ -0,0 +1,177 @@
+.cmp-new-footer{
+    background-color: $color-grey-silver;
+    padding: 0 24px;
+
+    &__main-container{
+        max-width: toRem(1440);
+        margin: 0 auto;
+        padding: toRem(48) 0;
+    }
```

Comment:
Px versus rem is tricky, because it's not "always use one or the other" but more like "use where it makes sense."

In this case there are two `padding` declarations, one with rems, the other without, I think those should both use rems.

Then there's a `max-width` declaration that I think should **not** use rems, since the way `max-width` interacts with existing. The breakpoint mixins (`min-screen(...)` for example) will be using a px size defined in that file.

Since `max-width` will scale with user font/zoom settings, but px won't, that could lead to unexpected behavior.